comparators/added-website.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = addedWebsite;
+
+function addedWebsite(newVersion, oldVersion, callback) {
+  if (oldVersion && oldVersion.properties && newVersion && newVersion.properties && oldVersion.properties.website === newVersion.properties.website) {
+    callback(null, {});
+  } else if (newVersion && newVersion.properties && 'website' in newVersion.properties) {
+    callback(null, {'result:added-website': true});
+  } else if (oldVersion && oldVersion.properties && ('website' in oldVersion.properties) && newVersion && newVersion.properties && !('website' in newVersion.properties)) {
```

Comment:
@amishas157 this is checking a the website existed in the `oldVersion` but does not in the `newVersion`. The usecase is somebody removing a website from a feature.